plugins/spark/spark-env.sh.in

```diff
@@ -67,11 +67,17 @@
 # - MKL_NUM_THREADS=1        Disable multi-threading of Intel MKL
```

Comment:
Should this file be renamed?

Reply:
No, that's the file we're using as the base for the default spark-env.sh file that a user can modify.  It's staying spark-env.sh.  The one we generate using xslt's is changing names so it won't confuse end users.